.travis.yml

```diff
@@ -25,39 +29,25 @@ rust: &rust
     - LD_LIBRARY_PATH=$HOME/lib
     - PKG_CONFIG_PATH=$HOME/lib/pkgconfig
   before_install: # versions from https://github.com/erickt/rust-zmq/blob/master/.travis.yml
-    - wget https://github.com/jedisct1/libsodium/releases/download/1.0.16/libsodium-1.0.16.tar.gz
-    - tar xvfz libsodium-1.0.16.tar.gz
-    - cd libsodium-1.0.16 && ./configure --prefix=$HOME && make && make install && cd ..
-    - wget https://github.com/zeromq/libzmq/releases/download/v4.2.5/zeromq-4.2.5.tar.gz
-    - tar xvfz zeromq-4.2.5.tar.gz
-    - cd zeromq-4.2.5 && ./configure --prefix=$HOME --with-libsodium && make && make install && cd ..
+    - ./install.sh
 
   script:
-  - cargo clean
-  - cargo build
-  - cargo test
-  - |
-    if [[ "$TRAVIS_RUST_VERSION" == stable ]]; then
-      (rustfmt --version || rustup component add rustfmt) &&
-      cargo fmt -- --check --color=auto &&
-      (cargo-clippy --version || rustup component add clippy) &&
-      cargo clippy -- -D warnings &&
-      (cargo-audit -h || cargo install cargo-audit) &&
-      cargo audit       
-    fi
+    - cargo clean
+    - cargo build
+    - cargo test
+    - |
+      if [[ "$TRAVIS_RUST_VERSION" == stable ]]; then
+        (rustfmt --version || rustup component add rustfmt) &&
+        cargo fmt -- --check --color=auto &&
+        (cargo-clippy --version || rustup component add clippy) &&
+        cargo clippy -- -D warnings &&
+        (cargo-audit -h || cargo install cargo-audit) &&
+        cargo audit       
+      fi
 
   after_success: |
     if [[ "$TRAVIS_RUST_VERSION" == nightly ]]; then
-      wget https://github.com/SimonKagstrom/kcov/archive/master.tar.gz &&
-      tar xzf master.tar.gz &&
-      cd kcov-master &&
-      mkdir build &&
-      cd build &&
-      cmake .. &&
-      make &&
-      sudo make install &&
-      cd ../.. &&
-      rm -rf kcov-master &&  
+      ./install2.sh 
```

Comment:
The Travis document becomes more clean after extracting installation steps to separate files now.

It will be better if the installation script has a more meaningful name so that it can reveal its intention.

Reply:
ok